loader/src/injector/hook.cpp

```diff
@@ -850,15 +819,21 @@ static void hook_unloader() {
     ino_t art_inode = 0;
     dev_t art_dev = 0;
 
-    for (auto &map : lsplt::MapInfo::Scan()) {
+    for (auto &map : cached_map_infos) {
```

Comment:
Above the parent function, please add a comment explaining why we do cache, especially here.